driving_school/data/mail_template.xml

```diff
@@ -0,0 +1,61 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<odoo noupdate="1">
+  <record id="mail_template_43" model="mail.template">
+    <field name="name">URLs Driving C</field>
+    <field name="model_id" ref="base.model_res_partner"/>
+    <field name="subject">Your driving lesson booking links</field>
+    <field name="body_html"><![CDATA[
+        <div style="margin: 0px; padding: 0px;" data-oe-version="2.0">
+        Dear <t t-out="object.name or ''" />
+        <br />
+        <br />
+        Please find herein the URLs to book your lessons: <br />
+        <a href="https://250619-drivingschool.odoo.com/book/60df3809"
```

Comment:
Will this URL work? When module will be used in any customer's db?

Reply:
No I think. Instead of UR of **https://250619-drivingschool.odoo.com/** , URL of current db should be set with correct Id of book.